src/clientsettingsdlg.h

```diff
@@ -69,6 +70,17 @@ class CClientSettingsDlg : public CBaseDlg, private Ui_CClientSettingsDlgBase
     void    UpdateAudioFaderSlider();
     QString GenSndCrdBufferDelayString ( const int iFrameSize, const QString strAddText = "" );
 
+    // Define strings for launching System Sound Setup
+#ifdef _WIN32
+    QString OpenSoundSettingsCmd = "control mmsys.cpl,,1";
+#elif defined( Q_OS_MACX )
+    QString OpenSoundSettingsCmd = "open /System/Library/PreferencePanes/Sound.prefPane";
+#elif defined( __linux__ )
+    QString OpenSoundSettingsCmd = "gnome-control-center sound";
```

Comment:
What if I'm not running Gnome?  In fact, what if I'm running a remote X server with JACK netaudio out to the local PC?

Reply:
> What if I'm not running Gnome?

Really not my area of expertise, this is the only thing I could find to launch a dialog to get the sound settings display.
No experience with a remote Jamulus, would you then also need access to the local built sound settings?
Due to the many different use cases of Linux, perhaps we should not have this button for Linux but just for macOS and Windows.

Reply:
Ok. I think we should open qjackctl anyways...

Reply:
> qjackctl

Qjackctl is not really System Sound Settings, is it? Is it really needed to have a button to open qjackctl? Perhaps Qjackctl should be linked to the ASIO settings button? Obvious the button text should be updated as well for that to be correct.

Reply:
The problem is that this is sound system specific and we're trying to do the change in the generic part of the sound code.

Ideally that whole panel would be managed per sound system and the generic code would know nothing about it.

We definitely should _not_ try to shoe-horn in mixed concepts like opening qJackCtl with the ASIO Setting button!

Reply:
I have update the PR to be just for Windows and macOS now.